OpenRA.Mods.Common/Traits/SupportPowers/NukePower.cs

```diff
@@ -148,9 +148,6 @@ public override void Activate(Actor self, Order order, SupportPowerManager manag
 
 		public void Activate(Actor self, WPos targetPosition)
 		{
-			foreach (var launchpad in self.TraitsImplementing<INotifyNuke>())
-				launchpad.Launching(self);
```

Comment:
I'm not sure how the Lua scripting interface should handle launching, but replacing this is more consistent with skipping the launch sounds and any such changes can reasonably be considered out of scope.

Reply:
Hmm, damn, I thought it would trigger
```csharp
public override void Activate(Actor self, Order order, SupportPowerManager manager)
{
			base.Activate(self, order, manager);
```
 and nothing would change here, but you are right. This is not the case.